src/components/app/details/appConfig/AppComposeRouter.tsx

```diff
@@ -211,6 +227,7 @@ export default function AppComposeRouter({
                         )}
                     />,
                 ]}
+                <Redirect to={`${maximumAllowedUrl}`} />
```

Comment:
Can we write it as to={maximumAllowedUrl}. Also can we rename this?

Reply:
done